pkg/cloudprovider/provider/digitalocean/provider.go

```diff
@@ -28,13 +28,14 @@ import (
 )
 
 type provider struct {
+	// TODO(p0lyn0mial): remove privateKey
```

Comment:
As mentioned in the todo, please move it to a local variable within the Create function.

For simplicity we can also always create a ssh key. Regardless if coreos or something else

Reply:
yes, I'll issue a separate pull (step 2) for that, is that okay ?